echopype/utils/uwa.py

```diff
@@ -56,73 +64,91 @@ def calc_sound_speed(
 
 def calc_absorption(
     frequency,
-    distance=1000,
     temperature=27,
     salinity=35,
     pressure=10,
     pH=8.1,
     formula_source="AM",
 ):
-    """Calculate sea absorption in dB/m
+    """
+    Calculate sea water absorption in units [dB/m].
 
     Parameters
     ----------
-    frequency : int or numpy array
-        frequency in Hz
-    distance : num
-        distance in m (FG formula only)
-    temperature : num
-        temperature in deg C
-    salinity : num
-        salinity in ppt
-    pressure : num
-        pressure in dbars
-    pH : num
+    frequency: int or numpy array
+        frequency [Hz]
+    temperature: num
+        temperature [deg C]
+    salinity: num
+        salinity [PSU, part per thousand]
+    pressure: num
+        pressure [dbars]
+    pH: num
         pH of water
-    formula_source : str
-        Source of formula used for calculating sound speed.
-        Default is to use Ainlie and McColm (1998) (``formula_source='AM'``).
-        Another option is to the formula supplied by AZFP (``formula_source='AZFP'``).
-        Another option is to use Francois and Garrison (1982)
-        supplied by ``arlpy`` (``formula_source='FG'``).
+    formula_source: str, {"AM", "FG", "AZFP"}
+        Source of formula used to calculate sound speed.
+        "AM" uses the formula from Ainslie and McColm (1998).
+        "FG" uses the formula from Francois and Garrison (1982).
+        "AZFP" uses the the formula supplied in the AZFP Matlab code.
+        See Notes below for the references.
 
     Returns
     -------
-    Sea absorption [dB/m]
+    Seaw ater absorption [dB/m].
+
+    Notes
+    -----
+    Ainslie MA, McColm JG. (1998). A simplified formula for viscous
+    and chemical absorption in sea water.
+    The Journal of the Acoustical Society of America, 103(3), 1671–1672.
+    https://doi.org/10.1121/1.421258
+
+    Francois RE, Garrison GR. (1982). Sound absorption based on
+    ocean measurements. Part II: Boric acid contribution and equation
+    for total absorption.
+    The Journal of the Acoustical Society of America, 72(6), 1879–1890.
+    https://doi.org/10.1121/1.388673
+
+    The accuracy of the simplified formula from Ainslie & McColm 1998
+    compared with the original complicated formula from Francois & Garrison 1982
+    was demostrated between 100 Hz and 1 MHz.
     """
-    # fmt: off
     if formula_source == "FG":
         f = frequency / 1000.0
-        d = distance / 1000.0
+        d = 1 / 1000.0  # ref to 1 m
```

Comment:
> # ref to 1 m
Expand a bit to make this clearer. "Referenced to 1 m"?

Reply:
Ended up removing `d` as it's just a scaling factor that can be applied at the end.